source/containers/Form/hooks/formActions.ts

```diff
@@ -312,7 +312,7 @@ export function validateAnswer(
  * @param onErrorCallback   Called when validation failed for an input field.
  * @param onValidCallback   Called when all field validated successfully.
  */
-export function validateAllStepAnswers( state: FormReducerState, onErrorCallback, onValidCallback ) {
+export function validateAllStepAnswers( state: FormReducerState, onErrorCallback: () => void, onValidCallback: () => void ) {
   // Collect all questions.
```

Comment:
?

Reply:
Actually I have no idea what this comment means either, so it should be removed. But again, not really a part of the PR...